network/wsNetwork.go

```diff
@@ -1895,15 +1895,14 @@ func (wn *WebsocketNetwork) getDNSAddrs(dnsBootstrap string) (relaysAddresses []
 		}
 		relaysAddresses = nil
 	}
-	if wn.config.EnableCatchupFromArchiveServers || wn.config.EnableBlockServiceFallbackToArchiver {
-		archiverAddresses, err = wn.resolveSRVRecords("archive", "tcp", dnsBootstrap, wn.config.FallbackDNSResolverAddress, wn.config.DNSSecuritySRVEnforced())
-		if err != nil {
-			// only log this warning on testnet or devnet
-			if wn.NetworkID == config.Devnet || wn.NetworkID == config.Testnet {
-				wn.log.Warnf("Cannot lookup archive SRV record for %s: %v", dnsBootstrap, err)
-			}
-			archiverAddresses = nil
+
+	archiverAddresses, err = wn.resolveSRVRecords("archive", "tcp", dnsBootstrap, wn.config.FallbackDNSResolverAddress, wn.config.DNSSecuritySRVEnforced())
+	if err != nil {
```

Comment:
this is not under a config flag anymore. IMO this should not be merged until we have some archive DNS. Even if it has just a few relays nodes (assuming they are all archival at the moment).
Peer selector should handle this OK but it is worth testing catchup on this branch on mainnet to ensure it does not stuck on non-existing `archive` DNS requested by peer selector

Reply:
This was enabled by default until #5706  from September was merged. I was tired of seeing the warning in testnet about it not being set. The archive dns never existed on mainnet and it ran fine (returning an error, which results in the `archiveAddresses` variable being set to nil.